src/Bookings.js

```diff
@@ -1,9 +1,10 @@
-import React from "react";
+import React, { useState } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState(FakeBookings);
```

Comment:
well done for setting the initial state using fakeBookings.json. Remember only React components should begin with a capital letter, so line 4 should be
```
import fakeBookings from "./data/fakeBookings.json";
```